.github/workflows/build-wheel-linux.yml

```diff
@@ -0,0 +1,91 @@
+# (C) Copyright 2024- ECMWF.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+
+name: Build Python Wheel for Linux
+
+on:
+  # Trigger the workflow manually
+  workflow_dispatch: ~
+
+  # Allow to be called from another workflow
+  workflow_call: ~
+
+  # TODO automation trigger
+
+jobs:
+
+  build:
+
+    runs-on: [self-hosted, Linux, platform-builder-Rocky-8.6]
+    # TODO which manylinux do we want to build for? 2014? 2_28? 2_34? Matrix?
+    container: wheelmaker_2_28:0.1
+
+    name: Build manylinux_2_28
+
+    steps:
+      # TODO which project do we build odc wheel from -- this, or some bundle? Or can we obtain the compiled eckit artifact somehow?
+      - run: git clone --branch develop --depth=1 https://github.com/ecmwf/eckit.git /src/eckit
```

Comment:
I would say we use the same branch -- typically it would be master building against master.
Or develop building against develop.